mzLib/Proteomics/RetentionTimePrediction/Chronologer/ChronologerRetentionTimeEstimator.cs

```diff
@@ -0,0 +1,125 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using TorchSharp;
+
+namespace Proteomics.RetentionTimePrediction.Chronologer;
+public static class ChronologerRetentionTimeEstimator
+{
+    /// <summary>
+    /// Uses the Chronologer model to predict the retention time of a peptide.
+    /// Only modifications present in the Chronologer dictionary are supported.
+    /// </summary>
+    /// <param name="baseSequence"></param>
+    /// <param name="fullPeptideSequence"></param>
+    /// <returns></returns>
```

Comment:
What form should the mods take? What is returned, i.e. seconds, minutes, hydrophobicity index?

Reply:
You can use the same format for the strings. For example, from the PsmFromTsv object you can use both the base sequence and the full sequence properties.